core-bundle/src/Filesystem/Dbafs.php

```diff
@@ -52,6 +52,7 @@ class Dbafs implements ResetInterface, DbafsInterface
     private string $dbPathPrefix = '';
     private int $maxFileSize = 2147483648; // 2 GiB
     private int $bulkInsertSize = 100;
+    private bool $useLastModified = false;
```

Comment:
Again, I'd propose to make that `true` by default.

Reply:
see https://github.com/contao/contao/pull/3774/commits/e508cfca00293922378cab8189c12d8e3469e141